covidsafe.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.example.corona.comms";
+option java_outer_classname = "TestProto";
+option objc_class_prefix = "CC";
+
+package corona;
+
+service Registration {
+	rpc registerUser (RegistrationInfo) returns (Registered) {}
+}
+
+service Logging {
+	rpc sendInfectedLogs (stream Log) returns (AddedLogs) {}
+	rpc getBLTContactLogs (stream Key) returns (stream BLTContactLog) {}
+}
```

Comment:
We probably want to add the services into one single `service` definition instead of two, that way we need only one server binding attached to the services.